src/game_party.cpp

```diff
@@ -255,10 +255,6 @@ bool Game_Party::IsItemUsable(int item_id, const Game_Actor* target) const {
 		return false;
 	}
 
-	if (data.party.size() == 0) {
```

Comment:
Does removing of this check have any practical use? You can't open the item scene in the menu when the party is empty and there is no event command to use items.

Reply:
Maybe not, but having it has no practical use either. It wasn't in RPG_RT so I took it out.